src/client/OpenFin/apps/Launcher/NlpSuggestions/executions/RfqExecution/state.ts

```diff
@@ -75,7 +76,7 @@ const rfqExecutionState$: Observable<RfqNlpExecutionState> = nlpIntent$.pipe(
         exhaustMap(() =>
           createCreditRfq$({
             instrumentId: instrument.id,
-            quantity: notional,
+            quantity: notional * 1000,
```

Comment:
just cross-check with the non-NLP behaviour, please .. ideally, we should not be applying this x 1000 willy nilly wherever we kick off an RFQ .. if we can converge all points to one place where the conversion is done, then I'd be happier (not always possible, ofc)
UI treatments will probably be distinct e.g. NLP Are you sure? bit above vs RFQ Form where it has the (1000s) label as a reminder